CHANGELOG.md

```diff
@@ -16,14 +16,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Two new compiler flags: `--outputsAreSets` and `--ignoreOrder`
 - Experimental Snowflake sink
   ([#774](https://github.com/feldera/feldera/issues/774))
 - WebConsole: Snowflake output connector dialog and node
   ([#859](https://github.com/feldera/feldera/issues/859))
-
 - Source and sink connector documentation
   ([#882](https://github.com/feldera/feldera/pull/882))
+- Enforce distinct outputs.  This is equivalent to applying
+  `SELECT DISTINCT` to each output view.
+  ([#871](https://github.com/feldera/feldera/issues/871))
+- Ignore outermost `ORDER BY` clauses, which don't make
```

Comment:
this too (in /docs)

Reply:
These have already been documented in docs/contributors/compiler.md
I don't know why that file, it's very hard to find.

Reply:
Internal docs are not enough in this case.  I agree, this needs to be documented.  We should also document upsert and primary key behavior. Let's so a separate PR.

Reply:
The more important question is whether the UI breaks when these flags are not applied.
Perhaps the UI should expose them.

Reply:
>The more important question is whether the UI breaks when these flags are not applied. Perhaps the UI should expose them.

It will break, but this PR ensures that you won't be able to run programs where the flags aren't set on our service.

Reply:
These flags are not exposed to the user and are always applied.  I prefer to keep it this way.

Reply:
Then we are supporting a weird dialect of SQL as a service. That's dangerous.

Reply:
IMO exposing and supporting _multiple_ _different_ weird dialects is worse than just one of them

Reply:
I cannot write in the SQL documentation for the compiler that ORDER BY is ignored. 
That's a choice the compiler gives to the user.
We can have another document that explains how the service SQL is different from the Calcite SQL.

Reply:
Our SQL docs must document the behavior of Feldera, not the internals that are not exposed to the user.

Reply:
I am documenting the compiler, which is open source  and may be used for other purposes that running feldera pipelines. These should be as separate as possible.

Reply:
I think this can be solved by writing the sentences in the documentation accordingly. 
Also not all ORDER BYs are ignored even with this flag so the ORDER BY documentation / grammer description etc. will still be relevant.